src/main/groovy/edu/oregonstate/mist/invencycle/db/BikeDAO.groovy

```diff
@@ -11,6 +11,118 @@ import org.skife.jdbi.v2.sqlobject.Bind
 public interface BikeDAO extends Closeable {
 
     /**
+     * POST a bike
+     */
+    @SqlUpdate("""
+              INSERT INTO BRAKE (BRAKE_ID, BRAKE_MAKE, BRAKE_MODEL, BRAKE_ROTOR_SIZE, BRAKE_POSITION)
+              VALUES (BIKE_BRAKE_ID_SEQ.nextval, :brake_make_front, :brake_model_front, :brake_rotor_size_front, 'Front')
+               """)
+    void postBrakeFront(@Bind("brake_make_front") String brake_make_front,
+                        @Bind("brake_model_front") String brake_model_front,
+                        @Bind("brake_rotor_size_front") Integer brake_rotor_size_front)
+
```

Comment:
you should use camelCase to name your variables here. For example, using `brakeMakeFront` rather than`brake_make_front`